### PR TITLE
Use new issue templates for ddev

### PR DESCRIPTION
**Pull request by rfay**

### CODE REVIEW

**Comment by rickmanelius** [comment]
Cool. I didn't realize this was a thing. Linking for others that may be interested https://blog.github.com/2018-01-25-multiple-issue-and-pull-request-templates/

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```
Comment (rickmanelius) [inline .github/ISSUE_TEMPLATE/Bug_report.md]: The "alleged" bug ;)

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+** Version and configuration  information (please complete the following information):**
```
Comment (rickmanelius) [inline .github/ISSUE_TEMPLATE/Bug_report.md]: Nits. No space is " Version" and remove the double space.

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+ - Host computer OS and Version: [e.g. Windows 10, macOS High Sierra]
+ - Docker version information (use `docker version`) and insert it here.
+ - ddev version information (use `ddev version`)
+ - config.yaml contents for the misbehaving project
+ - Do you have any custom configuration (nginx, php, mysql) in the .ddev folder?
```
Comment (rickmanelius) [inline .github/ISSUE_TEMPLATE/Bug_report.md]: At some point, we may want to direct people to https://supportdetails.com/ for system info. Or have a more verbose version for `ddev version --bug-report` to find and spit all this info out.

File: .github/ISSUE_TEMPLATE/Custom.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+---
+name: Meta Issues
+about: This issue type is for major meta-issues or discussions
```
Comment (rickmanelius) [inline .github/ISSUE_TEMPLATE/Custom.md]: Should this filename be named "Meta.md"?

Comment (rickmanelius) [reply .github/ISSUE_TEMPLATE/Custom.md]: I recognize this issue might be self-explanatory, but I'm assuming some guidance on stating that this should reference two or more child issues with perhaps something like

* [ ] Child issue 1 - Link to issue
* [ ] Child issue 2 - Link to issue
* [ ] Child issue 3 - Link to issue

File: .github/ISSUE_TEMPLATE/Bug_report.md (hunk 3 above)
Comment (rfay) [reply .github/ISSUE_TEMPLATE/Bug_report.md]: I sure like that idea! `ddev version --bug-report`!

File: .github/ISSUE_TEMPLATE/Custom.md (hunk 4 above)
Comment (rfay) [reply .github/ISSUE_TEMPLATE/Custom.md]: Renamed to Meta, good call. Right now, you're the meta-master, so will leave this free-form for now.

**Pull request merged**